src/utils/RNPxlSearch.cpp

```diff
@@ -1639,7 +1639,7 @@ class RNPxlSearch :
     // read list of nucleotides that can directly cross-link
     // these are responsible for shifted fragment ions. Their fragment adducts thus determine which shifts will be observed on b-,a-,y-ions
     String can_cross_link = getStringOption_("RNPxl:can_cross_link");
-    for (auto c : can_cross_link) { can_xl_.insert(c); }
+    for (const auto& c : can_cross_link) { can_xl_.insert(c); }
```

Comment:
using a reference for a char is probably slower than just copying the char, but who knows what the compiler will do :)

Reply:
yes, but again the meaning is clearer and the compiler can figure it out. There is probably anyways a better way of doing this with `insert` and iterators.